src/client/apps/edit/components/content/sections/images/components/edit_image.tsx

```diff
@@ -1,20 +1,20 @@
 import { Artwork } from "@artsy/reaction/dist/Components/Publishing/Sections/Artwork"
 import { Image } from "@artsy/reaction/dist/Components/Publishing/Sections/Image"
 import { SectionData } from "@artsy/reaction/dist/Components/Publishing/Typings"
-import {
-  onChangeHero,
-  onChangeSection,
-} from "client/actions/edit/sectionActions"
-import { Paragraph } from "client/components/draft/paragraph/paragraph"
-import { RemoveButton } from "client/components/remove_button"
 import { clone, without } from "lodash"
 import React, { Component } from "react"
 import { connect } from "react-redux"
 import styled from "styled-components"
+import {
+  onChangeHero,
+  onChangeSection,
+} from "../../../../../../../actions/edit/sectionActions"
```

Comment:
Why are the paths needing to change here? Are they in different directories now?

Reply:
I see the location didn't change but the file type did.

Reply:
Yeah tsx files are not resolving absolute paths correctly during test runs, going to make a ticket to resolve this issue separately

Reply:
👍